app/models/ems_event.rb

```diff
@@ -338,6 +338,29 @@ def ems_refresh_target
     ext_management_system
   end
 
+  def src_container_replicator
+    entity = container_replicator_id && ContainerReplicator.find(container_replicator_id)
```

Comment:
Great catch!  Fixed.
(I knew `find_by!` vs `find_by` and believed wrongly `find` belongs to the nil camp...)